### PR TITLE
Calculate fees - Closes #260

### LINKED ISSUE #260: Calculate fees
### Description

Make sure the fees calculation is performed properly together with the SDK version 5 compatibility layer.

Add required minimal fees taken from node constants.

Make sure that additional conditions are met according to the specification below.

### Motivation

- This feature is required by the UI clients

### Acceptance Criteria

- Dynamic fee calculation is performed
- Dynamic fee retrieval is possible
- Gateway is updated with the API version 2

#### Endpoints

- HTTP `/api/v2/fees`
- RPC `get.fees`

#### Request parameters

No params required.

#### Response example

200 OK

```jsonc
{
  "data": {
    "feeEstimatePerByte": {
      "low": 0,
      "medium": 1000,
      "high": 2000
    },
    "baseFeeById": {
      "2:0": "1000000000"
    },
    "baseFeeByName": {
      "token:transfer": "1000000000"
    },
    "minFeePerByte": 1000,
  },
  "meta": {
    "lastUpdate": 123456789,
    "lastBlockHeight": 25,
    “lastBlockId”: 1354568
  },
  "links": {}
}

```

503 Service Unavailable
```jsonc
{
  "error": true,
  "message": "Service is not ready yet"
}

```

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?
This PR resolves #260 

### How was it solved?
- [x] Move the fee estimate computation logic to compatibility layer
- [x] Extend the `blocks` index to include the following properties:
  - [x] `size`
- [x] Extend the `transactions` index to include the following properties:
  - [x] `fee`
  - [x] `minFee`
  - [x] `size`
- [x] Add endpoints
  - [x] HTTP - `/api/v2/fees`
  - [x] RPC - `get.fees`
- [x] Perform the fee computation properly in conjunction with the SDK version 5 compatibility layer
- [x] Add required minimal fees taken from node constants
  - [x] `baseFeeById` identified by `moduleAssetId`
  - [x] `baseFeeByName` identified by `moduleAssetName`
  - [x] `minFeePerByte`
- [x] Ensure the response structure is as specified in #260 

### How was it tested?
Local

### CODE REVIEW

File: services/core/methods/controllers/fees.js [hunk 1]
```diff
@@ -24,6 +24,9 @@ const getEstimateFeeByte = async () => {
 	result.feeEstimatePerByte.low = response.low;
```
Comment (nagdahimanshu) [inline services/core/methods/controllers/fees.js:24]: Does it make sense to resolve  all properties for `result`  and `meta` at `core` level and pass it directly to controllers? (In that way, we can avoid adding any logic to controller)

For example: we can do something like  `const fees = {data: [], meta: {}}`  in `shared/core/dynamicFees`

**Review by nagdahimanshu — APPROVED** [review]
Looks good to me 👍

**Review by MichalTuleja — APPROVED** [review]
Great code, thanks @sameersubudhi

**Comment by sameersubudhi** [comment]
> Great code, thanks @sameersubudhi

Thank you 🙂

**Pull request merged**